draft-ietf-quic-recovery.md

```diff
@@ -318,12 +318,27 @@ is available, or if the network changes, the initial RTT SHOULD be set to 100ms.
 When an acknowledgement is received, a new RTT is computed and the timer
 SHOULD be set for twice the newly computed smoothed RTT.
 
-When crypto packets are sent, the sender SHOULD set a timer for the crypto
+When crypto packets are sent, the sender MUST set a timer for the crypto
 timeout period.  Upon timeout, the sender MUST retransmit all unacknowledged
```

Comment:
```suggestion
retransmission interval.  Upon timeout, the sender MUST retransmit all unacknowledged
```

Reply:
This doc uses timeout and timeout period fairly consistently, so I'd like to keep it as is.